CHANGELOG.md

```diff
@@ -3,12 +3,13 @@ vNext (Month Day, Year)
 
 **TODO Next Release:**
 - Update README & aws-sdk-rust CI for MSRV upgrade to 1.54
+- Timeouts for requests are now configurable. You can set separate timeouts for each individual request attempt and all attempts made for a request. (smithy-rs#831)
```

Comment:
This should go under New This Week rather than our todo list for cutting the release